docs/development/i18n.rst

```diff
@@ -178,10 +189,11 @@ result, it can be merged with:
 
 |Weblate commit Unlock|
 
-`Weblate`_ pushes the translations done via the web interface
-to the develop branch in a fork of the `SecureDrop git repository`_.
-These commits must be manually cherry-picked and proposed as pull
-requests for the `SecureDrop git repository`_.
+Translation can now begin. As translators make progress, `Weblate`_
+pushes the translations done via the web interface to the
+``securedrop-i18n`` fork of the `SecureDrop git repository`_. When the
```

Comment:
i think it's better to link to the fork itself here and not the securedrop repo. you could also mention the branch they are pushed to. i think i18n?